CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.0] - 2020-12-01
+
```

Comment:
I think we should have some notes here?

Reply:
should we maybe remove all the alpha and beta releases and copy the changelog entries in the final one? WDYT?

Reply:
Added a note.

Reply:
I guess it's fine to have beta's for the operator. In actual `release` notes / changelog we should have short list of changes between `12.1.2` - `13.0.0` but not the intermediate bug fixes.

Reply:
> I guess it's fine to have beta's for the operator. In actual release notes / changelog we should have short list of changes between 12.1.2 - 13.0.0 but not the intermediate bug fixes.

+1